apps/partner2/client/overview_layout_factory.coffee

```diff
@@ -16,7 +16,14 @@ articlesTemplate = -> require('../components/fixed_cells_count_carousel/articles
 
 module.exports = (partner, profile) ->
   contract =
-    institution: []
+    institution: [
+      galleryThreeHero partner, profile
+      galleryThreeAbout partner, profile
+      galleryThreeNews partner, profile
+      galleryThreeShows partner, profile
+      galleryThreeFairBooths partner, profile
```

Comment:
Looking at this, I would guess that Institutions should not get FairBooths tab. I might be wrong here...